### ISSUE TITLE
"Member" button in French interface displays in English - Should be "Membre"

### ISSUE THREAD

**Issue by PhilAndrecheck** [gctools-outilsgc/gccollab#333]
When in the French language interface of a group, the "Member" button is in English.
It should be "Membre" in French.

**Comment by klecuyer** [comment]
@PhilAndrecheck can you send me a link or screenshot to where this is

**Comment by Troy-Lawson** [comment]
I dont know what member button this refers too. Unable to replicate

**Comment by Troy-Lawson** [comment]
unable to replicate
